includes/compatibility/class-wcpdf-compatibility-third-party-plugins.php

```diff
@@ -143,6 +146,40 @@ public function add_product_bundles_classes ( $classes, $document_type, $order,
 		return $classes;
 	}
 
+
+	/**
+	 * WPC Product Bundles
+	 * @param string $classes       CSS classes for item row (tr) 
+	 * @param string $document_type PDF Document type
+	 * @param object $order         WC_Order order
+	 * @param int    $item_id       WooCommerce Item ID
+	 */
+	public function add_wpc_product_bundles_classes ( $classes, $document_type, $order, $item_id = '' ) {
```

Comment:
since the `$item_id` parameter is an `int`, the default value should be `0`, not `''` (you can fix this on the other functions connected to the same `wpo_wcpdf_item_row_class` too!)